pkg/cmd/provisioning/ibmcloud/create.go

```diff
@@ -0,0 +1,177 @@
+package ibmcloud
+
+import (
+	"fmt"
+	"io"
+	"io/ioutil"
+	"log"
+	"os"
+	"path/filepath"
+
+	credreqv1 "github.com/openshift/cloud-credential-operator/pkg/apis/cloudcredential/v1"
+	"github.com/openshift/cloud-credential-operator/pkg/cmd/provisioning"
```

Comment:
To keep consistency, these imports should come in a separate block at the end

Reply:
Done. Hopefully I understood you correctly.